lib/spandex_ecto/ecto_logger.ex

```diff
@@ -22,9 +25,11 @@ defmodule SpandexEcto.EctoLogger do
     tracer = config[:tracer] || raise "tracer is a required option for #{inspect(__MODULE__)}"
     service = config[:service] || :ecto
 
-    unless Application.get_env(otp_app, tracer)[:disabled?] do
+    trace_running? = match?({:ok, %Spandex.SpanContext{}}, tracer.current_context())
+    disabled? = Application.get_env(otp_app, tracer)[:disabled?]
+
+    if trace_running? and not disabled? do
```

Comment:
I think we could skip the `trace_running?` and `disabled?` above and just do:

```suggestion
    if tracer.current_trace_id() do
```

If tracing is disabled or there is no trace active, this will return `nil`.

Reply:
Yeah, good idea :+1: